### ISSUE TITLE
Get protocols to start maintaining this

### ISSUE THREAD

**Issue by JorikSchellekens** [keep-starknet-strange/awesome-starknet#53]
We need a twitter campaign / channel drive to get protocols submitting themselves to this repo

**Comment by rocki202** [comment]
campaign still going in

**Comment by Giri-Aayush** [comment]
was this campaign completed? was it a success?

**Comment by JorikSchellekens** [comment]
> was this campaign completed? was it a success?

No, just some tweets on my part. @Giri-Aayush you can push for this.
